lib/presentation/volunteer/accomodation/accomodation_list_page.dart

```diff
@@ -0,0 +1,189 @@
+          ),
+        ),
+      ),
+    );
+  }
+}
+
+class AccomodationListLoadSuccessWidget extends StatelessWidget {
+  const AccomodationListLoadSuccessWidget({
+    Key? key,
+    required this.accomodations,
+  }) : super(key: key);
+
+  final List<Accomodation> accomodations;
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.all(4.0),
+      child: ListView(
+        children: [
+          const Padding(
+            padding: EdgeInsets.fromLTRB(0, 14, 0, 14),
+            child: Text(
+              'Lista zgłoszonych mieszkań',
+              textAlign: TextAlign.center,
+            ),
+          ),
+          ...accomodations
+              .map(
+                (accomodation) => Padding(
+                  padding: const EdgeInsets.fromLTRB(0, 4, 0, 4),
+                  child: ListTile(
+                    tileColor: () {
+                      switch (accomodation.status) {
+                        case AccomodationVerificationStatus.phoneVerified:
+                          return Colors.yellow[100];
+                        case AccomodationVerificationStatus.inPersonVerified:
+                          return Colors.green[200];
+                        case AccomodationVerificationStatus.added:
+                        default:
+                          return Colors.grey[200];
+                      }
+                    }(),
+                    leading: () {
+                      switch (accomodation.status) {
+                        case AccomodationVerificationStatus.phoneVerified:
+                          return Stack(
+                            children: const [
+                              Icon(Icons.phone),
+                              Icon(
+                                Icons.check,
+                                color: Colors.green,
+                                size: 35,
+                              ),
+                            ],
+                          );
+                        case AccomodationVerificationStatus.inPersonVerified:
+                          return Stack(
+                            children: const [
+                              Icon(Icons.person),
+                              Icon(
+                                Icons.check,
+                                color: Colors.green,
+                                size: 35,
+                              ),
+                            ],
+                          );
+                        case AccomodationVerificationStatus.added:
+                        default:
+                          return const Icon(Icons.question_mark);
+                      }
+                    }(),
+                    title: Text(accomodation.addressLine),
+                    subtitle: Column(
+                      children: [
+                        Text(
+                          (accomodation.voivodeship ?? '') +
+                              ' ' +
```

Comment:
Jeśli zarówno voivodeship jak i city będą puste to sama spacja się pojawi.
Zazwyczaj jakiś placeholder się daje w podobnych przypadkach.

Reply:
Added placeholder text for each of these